src/vlinder/appreciate.py

```diff
@@ -181,3 +182,18 @@ def _apply_scenario_weights(self) -> None:
                 appreciation = self.output_dict[scenario][option]["decision_makers_option_appreciation"]
                 weighted_appreciation = appreciation * weight / total_weight
                 self.output_dict[scenario][option]["scenario_appreciations"] = weighted_appreciation
+
+    def _best_dmo_calculator(self) -> None:
+        """
+        This function calculates the sum of all weighted appreciations per DMO and return the highest sum
+        :return: An updated output_dict with the best DMO option based on weighted appreciation for all scenarios
+        """
+        for scenario in self.input_dict["scenarios"]:
```

Comment:
FYI this is the code in the tool right now, but I checked the outputs and they are the same so it works fine.

    def get_best_decision_makers_option(case_output):
    """
    Retrieves the decision maker's option with the highest appreciation for each scenario.

    Args:
        case_output (dict): A dictionary containing case output data with scenarios and their respective options

    Returns:
        dict: A dictionary where each scenario maps to the decision maker's option with the highest appreciation
    """

    best_options = {}
    for scenario, options in case_output.items():
        best_option = max(options, key=lambda x: x["decision_makers_option_appreciation"])
        best_options[scenario] = best_option["decision_makers_option"]

    return best_options